src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -91,6 +91,14 @@
             <span class="invalid-feedback"
                   *ngIf="bucketForm.showError('owner', frm, 'required')"
                   i18n>This field is required.</span>
+            <cd-alert-panel
+              type="info"
+              *ngIf="bucketForm.get('owner').disabled"
+              spacingClass="me-1 mt-1"
+              i18n>
+                The bucket is owned by an account. UI does not support changing
+                the ownership of bucket owned by an account.
+            </cd-alert-panel>
```

Comment:
Should we block the ability to change a user owned bucket to an account owned bucket?

Reply:
should we? since we don't know what are all the accounts it would be impossible now? I am afraid that'll follow with a lot of code changes and extending new APIs to support account listing. If the action itself is non destructive or not raising any warnings later on, we could leave them as it is for now?